packages/help-center/src/hooks/use-open-article-in-help-center.ts

```diff
@@ -0,0 +1,23 @@
+import { HelpCenter } from '@automattic/data-stores';
+import { useDispatch as useDataStoreDispatch } from '@wordpress/data';
+
+const HELP_CENTER_STORE = HelpCenter.register();
```

Comment:
We can use `import { HELP_CENTER_STORE } from '../stores';`! Or maybe you tried and it doesn't work?